sdcm/cluster.py

```diff
@@ -1335,14 +1335,13 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
             scylla_yaml_contents = pattern.sub('seeds: "{0}"'.format(seed_address),
                                                scylla_yaml_contents)
 
-            # NOTICE: the following configuration always have to use private_ip_address for multi-region to work
```

Comment:
I've put this comment there for a reason, this should never be change to anything else then private_ip_address.

Reply:
Why it doesn't work with public IP? I will test later.

Reply:
it can't work with public address, cause this address isn't configured on the network device, scylla won't even boot

Reply:
Maybe on GCE the networking is setup a bit differently, and it would work. in AWS it doesn't work with public ip

Reply:
Thanks for your reply, I didn't consider too much about both parts.